Fails to build for me on NixOS:

```
gcc -Wall -g -O2 -I. -I/nix/store/yykmsrs5gnzhcihlrpawbhwamyvhha13-htslib-1.10.2/include -I./lz4  -c -o sample.o sample.c
bam_fastq.c: In function 'copy_tag':
bam_fastq.c:267:15: warning: implicit declaration of function 'bam_aux_get_str'; did you mean 'bam_aux_get'? [-Wimplicit-function-declaration]
  267 |     int ret = bam_aux_get_str(rec, tag, linebuf);
      |               ^~~~~~~~~~~~~~~
      |               bam_aux_get
sam_view.c: In function 'main_samview':
sam_view.c:468:47: warning: implicit declaration of function 'fai_path'; did you mean 'fai_fetch'? [-Wimplicit-function-declaration]
  468 |     if (fn_fai == 0 && ga.reference) fn_fai = fai_path(ga.reference);
      |                                               ^~~~~~~~
      |                                               fai_fetch
sam_view.c:468:45: warning: assignment to 'char *' from 'int' makes pointer from integer without a cast [-Wint-conversion]
  468 |     if (fn_fai == 0 && ga.reference) fn_fai = fai_path(ga.reference);
      |                                             ^
sam_view.c:682:13: error: 'errno' undeclared (first use in this function)
  682 |             errno = 0;
      |             ^~~~~
sam_view.c:43:1: note: 'errno' is defined in header '<errno.h>'; did you forget to '#include <errno.h>'?
   42 | #include "bedidx.h"
  +++ |+#include <errno.h>
   43 |
sam_view.c:682:13: note: each undeclared identifier is reported only once for each function it appears in
  682 |             errno = 0;
      |             ^~~~~
bam_sort.c: In function 'seq_to_bam':
bam_sort.c:2163:9: warning: implicit declaration of function 'bam_set_seqi'; did you mean 'bam_get_seq'? [-Wimplicit-function-declaration]
 2163 |         bam_set_seqi(seq, i, seq_nt16_table[(unsigned char)buf[i]]);
      |         ^~~~~~~~~~~~
      |         bam_get_seq
make: *** [Makefile:135: sam_view.o] Error 1
make: *** Waiting for unfinished jobs....
builder for '/nix/store/9zppbpar10asqd4iv7fa4pj5vfb46s8l-samtools-1.11.drv' failed with exit code 2
error: build of '/nix/store/9zppbpar10asqd4iv7fa4pj5vfb46s8l-samtools-1.11.drv' failed
```

--- a/pkgs/applications/science/biology/lumpy/default.nix
+++ b/pkgs/applications/science/biology/lumpy/default.nix
@@ -7,13 +7,14 @@ let
 
 in stdenv.mkDerivation rec {
   pname = "lumpy";
-  version = "0.3.0";
+  version = "0.3.1";
 
   src = fetchFromGitHub {
     owner = "arq5x";
     repo = "lumpy-sv";
-    rev = version;
-    sha256 = "0azhzvmh9az9jcq0xprlzdz6w16azgszv4kshb903bwbnqirmk18";
+    rev = "v${version}";
+    sha256 = "0r71sg7qch8r6p6dw995znrqdj6q49hjdylhzbib2qmv8nvglhs9";
+    fetchSubmodules = true;
   };
 
   nativeBuildInputs = [ which ];
@@ -26,6 +27,8 @@ in stdenv.mkDerivation rec {
     # Use Nix htslib over bundled version
     sed -i 's/lumpy_filter: htslib/lumpy_filter:/' Makefile
     sed -i 's|../../lib/htslib/libhts.a|-lhts|' src/filter/Makefile
+    # Also make sure we use the includes from Nix's htslib
+    sed -i 's|../../lib/htslib/|${htslib}|' src/filter/Makefile
   '';
 
   # Upstream's makefile doesn't have an install target

--- a/pkgs/applications/science/biology/samtools/default.nix
+++ b/pkgs/applications/science/biology/samtools/default.nix
@@ -2,11 +2,11 @@
 
 stdenv.mkDerivation rec {
   pname = "samtools";
-  version = "1.10";
+  version = "1.11";
 
   src = fetchurl {
     url = "https://github.com/samtools/samtools/releases/download/${version}/${pname}-${version}.tar.bz2";
-    sha256 = "119ms0dpydw8dkh3zc4yyw9zhdzgv12px4l2kayigv31bpqcb7kv";
+    sha256 = "1dp5wknak4arnw5ghhif9mmljlfnw5bgm91wib7z0j8wdjywx0z2";
   };
 
   nativeBuildInputs = [ perl ];
@@ -29,6 +29,6 @@ stdenv.mkDerivation rec {
     license = licenses.mit;
     homepage = "http://www.htslib.org/";
     platforms = platforms.unix;
-    maintainers = [ maintainers.mimame ];
+    maintainers = with maintainers; [ mimame unode ];
   };
 }

--- a/pkgs/development/python-modules/pysam/default.nix
+++ b/pkgs/development/python-modules/pysam/default.nix
@@ -80,13 +80,19 @@ buildPythonPackage rec {
       --deselect tests/AlignmentFileHeader_test.py::TestHeaderBAM::test_header_content_is_as_expected \
       --deselect tests/AlignmentFileHeader_test.py::TestHeaderCRAM::test_dictionary_access_works \
       --deselect tests/AlignmentFileHeader_test.py::TestHeaderCRAM::test_header_content_is_as_expected \
-      --deselect tests/AlignmentFile_test.py::TestIO::testBAM2SAM \
-      --deselect tests/AlignmentFile_test.py::TestIO::testSAM2BAM \
-      --deselect tests/AlignmentFile_test.py::TestIO::testWriteUncompressedBAMFile \
       --deselect tests/AlignmentFile_test.py::TestDeNovoConstruction::testBAMWholeFile \
       --deselect tests/AlignmentFile_test.py::TestEmptyHeader::testEmptyHeader \
       --deselect tests/AlignmentFile_test.py::TestHeaderWithProgramOptions::testHeader \
+      --deselect tests/AlignmentFile_test.py::TestIO::testBAM2BAM \
+      --deselect tests/AlignmentFile_test.py::TestIO::testBAM2CRAM \
+      --deselect tests/AlignmentFile_test.py::TestIO::testBAM2SAM \
+      --deselect tests/AlignmentFile_test.py::TestIO::testFetchFromClosedFileObject \
+      --deselect tests/AlignmentFile_test.py::TestIO::testOpenFromFilename \
+      --deselect tests/AlignmentFile_test.py::TestIO::testSAM2BAM \
+      --deselect tests/AlignmentFile_test.py::TestIO::testWriteUncompressedBAMFile \
+      --deselect tests/AlignmentFile_test.py::TestIteratorRowAllBAM::testIterate \
       --deselect tests/StreamFiledescriptors_test.py::StreamTest::test_text_processing \
+      --deselect tests/compile_test.py::BAMTest::testCount \
       tests/
   '';
 

